cluster/vagrant/util.sh

```diff
@@ -210,6 +210,27 @@ function kube-up {
 }
 EOF
 
+   cat <<EOF >"${HOME}/.kubernetes_vagrant_kubeconfig"
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://${MASTER_IP}:443
+  name: vagrant
+contexts:
+- context:
+    cluster: vagrant
+    namespace: default
+    user: vagrant
+  name: vagrant
+current-context: "vagrant"
+kind: Config
+preferences: {}
+users:
+- name: vagrant
+  user:
+    auth-path: ${HOME}/.kubernetes_vagrant_auth
```

Comment:
Include the auth information directly into the kubeconfig file instead of referencing auth-path.